development/src/utility/hexagons.js

```diff
@@ -666,6 +645,36 @@ var HexGrid = Class.create( {
 		});
 	},
 
+	/* getHexLine(x, y, dir, flipped)
```

Comment:
my bad, looks like a typo in line 158 - `ox.x+fx` should be `o.x+fx`. If you want I can make the change in the next PR, or you can since it's simple and I'm not sure how to test it yet.

Did you say this code also exists in abilities.js? Where?